iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/serviceclient/RegistryManagerTests.java

```diff
@@ -456,35 +456,76 @@ public void apply_configuration_e2e() throws Exception
     @StandardTierHubOnlyTest
     @Test
     @ContinuousIntegrationTest
-    public void deviceCreationWithSecurityScope() throws IOException, InterruptedException, IotHubException, URISyntaxException
+    public void deviceCreationWithDeviceScope() throws IOException, InterruptedException, IotHubException, URISyntaxException
```

Comment:
I couldn't find a sample to update that shows how to create devices with the service client. Can someone point me to one so I can update it too?

Reply:
[Here you go!](https://github.com/Azure/azure-iot-sdk-java/blob/9102801a0cb34db9db0d149f690165a4a9af1190/service/iot-service-samples/device-manager-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/DeviceManagerSample.java#L45)